### PR TITLE
Fixes things applying to the lighting object that shouldn't.

### LINKED ISSUE #17937: Colorful reagent grenades colour DARKNESS
Excessive use of colorful_reagent generates tiles of just solid colour.

These are darkness tiles, you can toggle  their appearance on and off with Toggle Darkness.

With darkness on.
image

Darkness turned off.
image

### PR DESCRIPTION
**Pull request by MrsTonedOne**
WAKE ME UP INSIDE.

fixes #17937 (most likely, i haven't tested shit)

### CODE REVIEW

**Comment by coiax** [comment]
Should test if this fixes #17937 and say if it does

**Comment by MrStonedOne** [comment]
\>test

**Comment by Thunder12345** [comment]
Does this bring an end to coloursmoke colouring the lighting?

**Comment by MrStonedOne** [comment]
yes it does

File: code/__DEFINES/is_helpers.dm [hunk 1]
```diff
@@ -77,6 +77,9 @@
 
 #define ismovableatom(A) (istype(A, /atom/movable))
 
+#define isobj(thing) istype(thing, /obj) //override the byond proc because it returns true on childs of /atom/movable that aren't objs
```
Comment (RemieRichards) [inline code/__DEFINES/is_helpers.dm]: the word is children.

**Comment by ChangelingRain** [comment]
I hope nothing was relying on this use of isobj()

**Pull request merged**